grid_map_pcl/test/GridMapPclLoaderTest.cpp

```diff
@@ -6,89 +6,110 @@
  *      Institute: ETH Zurich, Robotic Systems Lab
  */
 
-#include <cstdlib>
 #include <gtest/gtest.h>
+#include <rclcpp/rclcpp.hpp>
+#include <rcutils/error_handling.h>
 
-#include "grid_map_pcl/GridMapPclLoader.hpp"
+#include <cstdlib>
+#include <memory>
+#include <string>
 
+#include "grid_map_pcl/GridMapPclLoader.hpp"
 #include "PointcloudCreator.hpp"
 #include "test_helpers.hpp"
 
-namespace grid_map {
-namespace grid_map_pcl_test {
-
-TEST(GridMapPclLoaderTest, FlatGroundRealDataset)
+namespace grid_map
+{
+namespace grid_map_pcl_test
 {
-  grid_map_pcl_test::setVerbosityLevel(ros::console::levels::Warn);
+class GridMapPclLoaderTest : public ::testing::Test
+{
+protected:
+  void SetUp() override
+  {
+    node = std::make_shared<rclcpp::Node>("GridMapPclLoaderTest");
+
+    // Changing the logging verbosity to WARN
+    auto ret = rcutils_logging_set_logger_level(
+      node->get_logger().get_name(), RCUTILS_LOG_SEVERITY_DEBUG);
+    if (ret != RCUTILS_RET_OK) {
+      RCLCPP_ERROR(
+        node->get_logger(), "Failed to change logging severity: %s",
+        rcutils_get_error_string().str);
+      rcutils_reset_error();
+    }
+  }
 
-  grid_map::GridMapPclLoader gridMapPclLoader;
+  rclcpp::Node::SharedPtr node;
```

Comment:
```suggestion
  rclcpp::Node::SharedPtr node_;
```

Reply:
I noticed that the logger of the node was the only thing used from the node so I swapped the node with a logger instead. Made sure to include an "_" at the end of the variable name.